crates/cubecl-opt/src/lib.rs

```diff
@@ -0,0 +1,359 @@
+use std::{
```

Comment:
I would add a description of the lib and how it can be used to optimize the cubecl ir.